docs/GettingStarted.md

```diff
@@ -23,13 +23,14 @@ We strongly suggest pinning the version of the library you deploy.
 The easiest way to get started with the tracing client is to instrument your web application. ``ddtrace`` gem
 provides auto instrumentation for the following web frameworks and libraries:
 
+* [Ruby on Rails](#label-Ruby+on+Rails)
+* [Sidekiq](#label-Sidekiq)
+* [Sinatra](#label-Sinatra)
+* [Rack](#label-Rack)
```

Comment:
changed the order so that Web frameworks are at the top of the list; if you prefer I can revert the order to an alphabetical one